app/services/status-images.js

```diff
@@ -27,12 +27,16 @@ export default Service.extend({
     }
 
     let slug = repo.get('slug');
-
+    let token = this.auth.assetToken;
```

Comment:
@nikolaTrichkovski 
any specific reason to change the type of this variable from `const` to `let`?
I think lets use `const` for this and the following `branchQuery` variables as these are not getting reassigned, what do you think?

Reply:
you are right, this is an oversight. thanks for the comment